src/firmware/setup-and-install.md

```diff
@@ -67,7 +67,7 @@ For Windows, you can download and install [Git for Windows](https://git-scm.com/
 
 Make sure you close any current projects you have open or open a new window before moving forward with these steps.
 
-1. Click the **Source Control** button, click on **Clone Repository** and enter: `https://github.com/SlimeVR/SlimeVR-Tracker-ESP.git`. If you're using experimental MPU+QMC5883L trackers, you would instead clone `https://github.com/deiteris/SlimeVR-Tracker-ESP.git`.
+1. Click the **Source Control** button, click on **Clone Repository** and enter: `https://github.com/SlimeVR/SlimeVR-Tracker-ESP.git`. If you're using experimental MPU+QMC5883L trackers, you would instead clone `https://github.com/deiteris/SlimeVR-Tracker-ESP.git`, and if you're using BMI270 trackers you would clone `https://github.com/l0ud/SlimeVR-Tracker-ESP-BMI270.git`.
```

Comment:
This feels like it makes the sentence too long, maybe it can be organized a little better?

Reply:
I've seen a few people in the diy channel clone the MPU+QMC fork even when they don't need it, so I agree and believe it can cause some confusion.

Reply:
The best way I found to solve this was to just add a step before, explaining how to choose a firmware version based on the hardware the user has.